### ISSUE TITLE
Published magazine story does not appear in English magazine landing page

### ISSUE THREAD

**Issue by NickHaggerty1** [equinor/energyvision#1552]
A story was created in Norwegian first and published which worked fine. Then the En text was added and published. 
The story can be found in both languages on the site. However only the Norwegian story displays in the magazine landing page.
The story has be unpublished and published with no success. Could be editor error but we cant see the issue

Story pages:
https://www.equinor.com/magazine/offshore-wind-adventure
https://www.equinor.com/no/magasin/startskuddet-har-gatt-for-norges-havvindeventyr

Landing page:
https://www.equinor.com/magazine

**Comment by NickHaggerty1** [comment]
@fernandolucchesi @SvSven Could you take a look at this one and les us know if there was an issue from editor side or? @meols fyi

**Comment by NickHaggerty1** [comment]
Additionally.I have published an new story in NO only (for now) and this story has not shown up on the magazine section page either. This was published on 24/3. Unpublished. Republished. Updated. republished. Still not showing.
https://www.equinor.com/no/magasin/rent-norsk-hydrogen-til-europa

The EN story will go out today (27/3) And I imagine this will be the same issues as above.

@meols @fernandolucchesi FYI
